plugins/modules/fusion_pg.py

```diff
@@ -160,7 +155,7 @@ def create_pg(module, fusion):
         )
         await_operation(fusion, op)
 
-    module.exit_json(changed=changed)
+    return True
```

Comment:
`module.exit_json()` is actually being run later in `main()`. This is actually the same design as in [#64](https://github.com/Pure-Storage-Ansible/Fusion-Collection/pull/64), due to the same fact that it needs to run update after create because create cannot set all properties. [func tests](https://github.com/jankodera-ps/Fusion-Collection/blob/388df1b5f7636353e5596fbe9d1ad93f9e6c4764/tests/functional/test_fusion_pg.py#L180) actually check `exit_json()` function is being run on all paths